document-portal/document-portal-fuse.c

```diff
@@ -2993,13 +3005,62 @@ xdp_fuse_statfs (fuse_req_t req,
     xdp_reply_err (op, req, errno);
 }
 
+static ssize_t
+xdp_fuse_set_host_path_xattr (XdpInode   *inode,
+                              const char *value,
+                              size_t      size)
+{
+  errno = EPERM;
+  return -1;
+}
+
+static ssize_t
+xdp_fuse_get_host_path_xattr (XdpInode *inode,
+                              char     *buf,
+                              size_t    size)
+{
+  g_autoptr(PermissionDbEntry) entry =
+    xdp_lookup_doc (inode->domain->doc_id);
+  const char *path = NULL;
+  size_t path_size;
+
+  if (!entry)
+    return ENODATA;
```

Comment:
Should you return `-ENODATA` here?

Reply:
Perhaps this is what the FUSE API expects, which in that case, please ignore :smile:.

Reply:
No, good catch, this should return -1 and set errno like in all other cases.

Reply:
I copied this from `xdp_fuse_getxattr` where I believe this already was a mistake. Yikes.

Reply:
No problem. Thanks for your work on desktop linux!